### ISSUE TITLE
About source code

### ISSUE THREAD

**Issue by SYSUHPK** [hrnr/m-explore#46]
Hello, I have a question when reading the source code. I wonder why you just use compose to merge them rather that  apply stitch to merge in mapMerging thread.  Is the result better?

**Comment by hrnr** [comment]
I don't exactly understand the suggestion. What do you mean by apply stitch? Are you suggesting to use seam estimation before composition?

**Comment by SYSUHPK** [comment]
Yes, that's what I want to say, it seems there is no such thing in the source code
